### PR TITLE
Added Dark mode Feature And Footer Section With Social Media Feature 

### PR DESCRIPTION
**Pull request by ankit071105**
# Light Mode 

Screenshot 2023-10-10 at 11 49 33 AM

# Dark Mode 
Screenshot 2023-10-10 at 11 49 37 AM

# Footer Section
Screenshot 2023-10-10 at 11 49 44 AM

### CODE REVIEW

**Comment by sadanandpai** [comment]
FMC is not there on linkedin, insta etc

**Comment by ankit071105** [comment]
@sadanandpai 
okay then i will remove both the social media links

File: vue/src/index.css [hunk 1]
```diff
@@ -20,7 +20,7 @@ body {
 }
 
 a {
-  color: blue;
+  color: rgb(40, 59, 117);
```
Comment (arpansaha13) [inline vue/src/index.css:23]: Undo this change too

File: vue/index.html
Comment (arpansaha13) [inline vue/index.html:1]: Undo all changes from this file. Make the formatting how it was before.

File: native/index.html
Comment (arpansaha13) [inline native/index.html:1]: Take the styles out in a separate file and then link that stylesheet in this file.
Maintain proper formatting and indentation.
Remove all extra blank lines.

File: index.html [hunk 2]
```diff
@@ -28,6 +31,10 @@
         window.location =
           "https://sadanandpai.github.io/frontend-mini-challenges/react/dist/";
       }
+
+
+
+
```
Comment (arpansaha13) [inline index.html:37]: Remove these extra blank lines

File: index.html [hunk 3]
```diff
@@ -16,8 +16,11 @@
       name="description"
       content="Frontend Mini Challenges is a collection of frontend interview coding challenges"
     />
-  </head>
-  <body>
+</head>
+<body>
```
Comment (arpansaha13) [inline index.html:20]: Please format the files. The indentation doesn't look proper here.

File: index.html [hunk 4]
```diff
@@ -16,8 +16,11 @@
...
+
+
+
```
Comment (arpansaha13) [inline index.html:23]: Remove these extra blank lines

**Review by arpansaha13 — CHANGES_REQUESTED** [review]

File: native/index.html [hunk 5]
```diff
@@ -52,7 +237,18 @@
...
+        <label for="switch" style="background-color: black; color: #080a0c;"></label>
+  </label>
+      <script>
+        const $html = document.querySelector("html");
+        const $checkbox = document.querySelector("#switch");
+        
+        $checkbox.addEventListener("change", function() {
+            $html.classList.toggle("light-mode");
+        });
+        </script>
```
Comment (arpansaha13) [inline native/index.html:251]: Also, take this script out and put it in a separate file. And then link that script in this file.

**Comment by arpansaha13** [comment]
You may also remove the footer because most of the links are not required and the GitHub link is already there on the navbar.

**Comment by arpansaha13** [comment]
You can use prettier for the formatting.
